docs/changes.xml

```diff
@@ -62,6 +62,12 @@ increased the applications' startup timeout.
 </para>
 </change>
 
+<change type="bugfix">
+<para>
+fixed contents of SCRIPT_NAME in Ruby to always be an empty string.
```

Comment:
I suggest "force SCRIPT_NAME in Ruby to always be an empty string".

Reply:
Probably s/force/forced/ for consistency (past time used in other places)?

Reply:
Hmm, I'd say no, because in `bugfix` we usually use past to refer to the issue that happened in the past, but this sentence refers to what happens after the fix.